```
In file included from /home/travis/build/probonopd/ESP8266HueEmulator/ESP8266HueEmulator/ESP8266HueEmulator.ino:11:0:
/home/travis/arduino_ide/libraries/NtpClient/src/NtpClientLib.h:105:25: fatal error: ESPAsyncUDP.h: No such file or directory
 #include <ESPAsyncUDP.h>
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,6 +10,7 @@ install:
   - git clone https://github.com/interactive-matter/aJson.git
   - git clone https://github.com/PaulStoffregen/Time.git
   - git clone https://github.com/gmag11/NtpClient.git
+  - git clone https://github.com/me-no-dev/ESPAsyncUDP.git
   - sed -i -e 's|#define PRINT_BUFFER_LEN 256|#define PRINT_BUFFER_LEN 4096|g'  aJson/aJSON.h
   - cd -
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,11 +6,16 @@ before_install:
 install:
   - mkdir -p $HOME/arduino_ide/libraries/
   - cd $HOME/arduino_ide/libraries/
-  - git clone --branch 2.1.4 https://github.com/Makuna/NeoPixelBus.git
-  - git clone https://github.com/interactive-matter/aJson.git
-  - git clone https://github.com/PaulStoffregen/Time.git
-  - git clone https://github.com/gmag11/NtpClient.git
-  - git clone https://github.com/me-no-dev/ESPAsyncUDP.git
+  - git clone -o cf7b4a6 https://github.com/Makuna/NeoPixelBus
+  - git clone -o ec56354 https://github.com/interactive-matter/aJson
+  - cd aJson
+  - curl -OSLs https://gitlab.com/xarduino/lightsw/raw/2424e3a9348e7144373311af132ced38f14489ba/patch/ajson-void-flush.patch
+  - git apply ajson-void-flush.patch
+  - rm ajson-void-flush.patch
+  - cd ..
+  - git clone -o b05d455 https://github.com/PaulStoffregen/Time
+  - git clone -o 3c22f2f https://github.com/gmag11/NtpClient
+  - git clone -o b592ac6 https://github.com/me-no-dev/ESPAsyncUDP
   - sed -i -e 's|#define PRINT_BUFFER_LEN 256|#define PRINT_BUFFER_LEN 4096|g'  aJson/aJSON.h
   - cd -
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -19,6 +19,6 @@ install:
   - sed -i -e 's|#define PRINT_BUFFER_LEN 256|#define PRINT_BUFFER_LEN 4096|g'  aJson/aJSON.h
   - cd -
 script:
-  - sed -i -e 's|#include "/secrets.h"|//#include "/secrets.h"|g' ESP8266HueEmulator/ESP8266HueEmulator.ino
-  - sed -i -e 's|//const char|const char|g' ESP8266HueEmulator/ESP8266HueEmulator.ino
+  - sed -i -e 's|#include "/secrets.h"|//#include "/secrets.h"|g' ESP8266HueEmulator.ino
+  - sed -i -e 's|//const char|const char|g' ESP8266HueEmulator.ino
   - build_platform esp8266

